### PR TITLE
Net.shrew.ike.qikea

### PR DESCRIPTION
**Pull request by BeardOverflow**

### CODE REVIEW

**Comment by BeardOverflow** [comment]
bot, build net.shrew.ike.qikea

File: net.shrew.ike.qikea.json
Comment (barthalion) [inline net.shrew.ike.qikea.json:4]: Any reason you are such an old runtime? The latest is 20.08.

**Comment by BeardOverflow** [comment]
bot, build net.shrew.ike.qikea

File: net.shrew.ike.qikea.json
Comment (barthalion) [inline net.shrew.ike.qikea.json:14]: You should add `cleanup` array (global or per module) to remove static libraries and development headers.

File: net.shrew.ike.qikea.json
Comment (barthalion) [inline net.shrew.ike.qikea.json:8]: Does it really need host access?

File: net.shrew.ike.qikea.json
Comment (barthalion) [inline net.shrew.ike.qikea.json:52]: Does the license allow redistribution? If not, it should use `extra-data`.

**Review by barthalion — CHANGES_REQUESTED** [review]
Appdata fails validation:

```
builddir/files/share/appdata/net.shrew.ike.qikea.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
```

**Comment by barthalion** [comment]
Have you tried to contact the vendor about publishing their app on Flathub?

File: net.shrew.ike.qikea.json
Comment (BeardOverflow) [reply net.shrew.ike.qikea.json:4]: Yes, because I need an old version of openssl. The maximum compatible version is 1.0.2

File: net.shrew.ike.qikea.json
Comment (BeardOverflow) [reply net.shrew.ike.qikea.json:14]: Ok, I will review if any artifact is present after building

File: net.shrew.ike.qikea.json
Comment (BeardOverflow) [reply net.shrew.ike.qikea.json:8]: Not really, home access would be enough. Should I change it then?

File: net.shrew.ike.qikea.json
Comment (BeardOverflow) [reply net.shrew.ike.qikea.json:52]: Yes, the owner allows to do redistribution in its license. Is there any metadata for pointing it, apart of project_license?

Comment (barthalion) [reply net.shrew.ike.qikea.json:52]: No, just making sure. :+1:

File: net.shrew.ike.qikea.json
Comment (barthalion) [reply net.shrew.ike.qikea.json:8]: Yes please.

**Comment by BeardOverflow** [comment]
No, I haven't tried. Should I do it?

File: net.shrew.ike.qikea.json
Comment (barthalion) [reply net.shrew.ike.qikea.json:4]: I think it would be better to use 20.08 and bundle openssl 1.0 in the app.

**Comment by barthalion** [comment]
Yes, in general we expect submitters to reach out, as developers may be interested in helping or at least having write access to the repo.

**Comment by BeardOverflow** [comment]
I have written an email to the lead developer of qikea (ike), who is also employee of Shrew Soft Inc. I am waiting his answer now.

File: net.shrew.ike.qikea.json
Comment (BeardOverflow) [reply net.shrew.ike.qikea.json]: Done with a5d6b3c

**Comment by BeardOverflow** [comment]
bot, build net.shrew.ike.qikea

File: net.shrew.ike.qikea.json
Comment (BeardOverflow) [reply net.shrew.ike.qikea.json:4]: Ok, I am working about it

Comment (BeardOverflow) [reply net.shrew.ike.qikea.json]: Done with c45464f

File: net.shrew.ike.qikea.json
Comment (BeardOverflow) [reply net.shrew.ike.qikea.json:14]: Done with 92cbc7c

**Comment by BeardOverflow** [comment]
bot, build net.shrew.ike.qikea

**Comment by BeardOverflow** [comment]
bot, build net.shrew.ike.qikea

**Comment by BeardOverflow** [comment]
Ready! No more pending changes

P.S:

> I have written an email to the lead developer of qikea (ike), who is also employee of Shrew Soft Inc. I am waiting his answer now.

And about this, I have not received any answer still

**Comment by nedrichards** [comment]
OK, we will give them a little longer to respond.

**Comment by BeardOverflow** [comment]
bot, build net.shrew.ike.qikea

**Comment by BeardOverflow** [comment]
I did some additional patchs for improving the gui when dark theme is used and also renewed the screenshots

**Comment by barthalion** [comment]
Looks good. Let's wait till Monday for the answer.

**Comment by BeardOverflow** [comment]
No answer received

**Pull request closed without merging**
